### PR TITLE
rpc: protocol version

### LINKED ISSUE #574: Hardcoded Protocol Version
Acceptance Criteria: Hard code protocol version until we fix build issues. Create follow up issue to fix build issues.

Note: issue with Cgo during build process when importing Ethereum package 

The protocol version is currently hardcoded on the version package.
```go
// ProtocolVersion is the supported Ethereum protocol version (e.g., Homestead, Olympic, etc.)
const ProtocolVersion = 63
```

The current supported [protocol versions](https://github.com/ethereum/devp2p/blob/master/caps/eth.md) from geth are:

```go
// Constants to match up protocol versions and messages
const (
	eth63 = 63
	eth64 = 64
	eth65 = 65
)
```

### PR DESCRIPTION
**Pull request by fedekunze**
closes #574

### CODE REVIEW

File: rpc/eth_api.go [hunk 1]
```diff
@@ -29,6 +28,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/eth"
```
Comment (fedekunze) [inline rpc/eth_api.go]: so it seems that using this library is causing some issues when trying to build Ethermint. Can you take a look @araskachoi 🙏 ?

Comment (araskachoi) [reply rpc/eth_api.go:31]: hmm.. weird how it breaks when building in docker but works when you build binaries manually. Will keep digging tho

Comment (araskachoi) [reply rpc/eth_api.go]: @fedekunze can we try to merge the refactored code that we merged and see if that fixes this?

**Comment by fedekunze** [comment]
this is R4R again @araskachoi @noot

**Review by noot — APPROVED** [review]

File: types/protocol.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package types
+
+// Constants to match up protocol versions and messages
+const (
+	eth63 = 63
+	eth64 = 64
```
Comment (noot) [inline types/protocol.go]: in what cases would these 2 be used? looks like eth65 is the only one used rn

Comment (fedekunze) [reply types/protocol.go]: Good question. Idk, I just copied them over. Do you think we should remove them?

Comment (noot) [reply types/protocol.go]: if they aren't used we can probably remove them for now

**Pull request merged**